### PR TITLE
Added log_sigmoid function

### PR DESCRIPTION
**Pull request by V-G-spec**

### CODE REVIEW

File: .idea/ivy.iml
Comment (jkeane508) [inline .idea/ivy.iml:4]: Can your changes to this file please

File: .idea/misc.xml
Comment (jkeane508) [inline .idea/misc.xml]: This change needs to be removed aswell

File: ivy/functional/frontends/tensorflow/math.py
Comment (jkeane508) [inline ivy/functional/frontends/tensorflow/math.py:49]: Why is this a comment?

File: ivy/functional/frontends/tensorflow/math.py
Comment (jkeane508) [inline ivy/functional/frontends/tensorflow/math.py:51]: For lint a new line is required at the end of the file

File: ivy/functional/ivy/statistical.py
Comment (jkeane508) [inline ivy/functional/ivy/statistical.py:1]: Can you remove this change :)

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:332]: Three new lines will throw a lint error we only need to newlines between functions and tests

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Some changes to formatting needed :)

File: ivy/functional/frontends/tensorflow/math.py
Comment (V-G-spec) [reply ivy/functional/frontends/tensorflow/math.py:49]: I was trying the commented-out way first but saw in the documentation that it is implemented using softplus which is more robust. Forgot to delete the commented out code, will do that right now

**Review by jkeane508 — CHANGES_REQUESTED** [review]
There seems to be some merge conflicts, can you please resolve :)

**Comment by V-G-spec** [comment]
I am not sure what the issue is. I tried changing ivy.iml and misc.xml files by removing the newlines at the end, but it still reverts back to the original file. Let me try few other things

**Comment by V-G-spec** [comment]
I think it's done now. I am not sure how to resolve conflicts with the files I made major changes in. Do I remove my changes, fetch the latest version, add my code, and then push?

**Comment by jkeane508** [comment]
You can usually fetch from the ivy master branch locally and you should be able to resolve conflicts there

**Comment by jkeane508** [comment]
Hi,

As you can see from the tests there are some lint errors and test errors, these need to be passing before we can merge. If you can make the relevant fixes that would be great!

**Comment by V-G-spec** [comment]
I raised this query on Discord channel as well, I am not sure where exactly some very specific test cases are failing. If you can suggest some points I should review, that would be great. Otherwise, I will look into it shortly. Thanks!

**Comment by V-G-spec** [comment]
Please correct me if I am wrong, but it seems to me that the errors are not because of my addition to the code but because of some other functions

**Pull request merged**
